auditok/cmdline.py

```diff
@@ -576,6 +576,8 @@ def main(argv=None):
         group.add_option("-r", "--rate", dest="sampling_rate", help="Sampling rate of audio data [default: %default]", type=int, default=16000, metavar="INT")
         group.add_option("-c", "--channels", dest="channels", help="Number of channels of audio data [default: %default]", type=int, default=1, metavar="INT")
         group.add_option("-w", "--width", dest="sample_width", help="Number of bytes per audio sample [default: %default]", type=int, default=2, metavar="INT")
+        group.add_option("-A", "--audio-device", dest="input_device_index", help="Audio device index", type=int, default=0, metavar="INT")
```

Comment:
Thank you for this!

I would use `-I`, `--input-device-index` for this option.

Also, following the [behavior of pyaudio](https://github.com/jleb/pyaudio/blob/master/src/pyaudio.py#L318), the default value for this argument should be `None`. This will ensure backward compatibility of `auditok` for users who don't specify this argument (just figured out that the default device is not always 0)

Reply:
Fixed!